app/views/sequences/show.html.haml

```diff
@@ -42,16 +42,18 @@
   .activities
     - @sequence.lightweight_activities_sequences.each do |las|
       - activity = las.lightweight_activity
+      - activity_path_params = @sequence_run ? [:run_key => @sequence_run.run_for_activity(activity).key] : []
```

Comment:
What is the case where there isn't a @sequence_run?

Reply:
Without that the teacher edition specs broke in this build:

https://travis-ci.org/concord-consortium/lara/builds/517974979

the first failed spec is here:

https://github.com/concord-consortium/lara/blob/master/spec/features/run_complete_sequence_spec.rb#L109

Reply:
So that seems like it would be fixed if the sequence show page does a redirect like the activity show page.